examples/with-typescript-graphql/package.json

```diff
@@ -1,42 +1,39 @@
 {
   "private": true,
   "scripts": {
-    "codegen": "graphql-let",
+    "codegen": "graphql-codegen",
```

Comment:
Hi, seems the `codegen` step is failing with: 

```sh
Something went wrong Failed to load schema for "./lib/resolvers-types.ts"
        Failed to load schema from pages/api/graphql.ts:

        Unable to load from file "/Users/ijjk/dev/vercel/next.js/examples/with-typescript-graphql/pages/api/graphql.ts": /Users/ijjk/dev/vercel/next.js/examples/with-typescript-graphql/pages/api/graphql.ts:1
import { createServer } from '@graphql-yoga/node'
^^^^^^

SyntaxError: Cannot use import statement outside a module
```

Reply:
@ijjk, indeed, it was broken, thanks for the heads-up!

Fixed in 2c16df4192900781f5b3c864340b0296356aa219